### PR TITLE
bb views 1 update. mapping update. fix angular authentication lesson plan typos

### PR DESCRIPTION
**Pull request by amaseda**

### CODE REVIEW

File: 11-multi-track/bb_views_part1/readme.md [hunk 1]
```diff
@@ -87,7 +94,7 @@ approach for our collection views later.
 Define a view in `js/backbone/views/grumble.js`. You should define the view such
```
Comment (amaseda) [inline 11-multi-track/bb_views_part1/readme.md]: Show how to implement `tagName` and `className` in `The el in Backbone` section.

File: 11-multi-track/bb_views_part1/readme.md [hunk 2]
```diff
@@ -111,14 +118,14 @@ Just like in vanilla JS, we usually want to render HTML and put it into the
 `el`. This method could be called anything, but convention is to call it
 `render`.
 
-### Exercise: Define a basic render method (10 minutes)
+### Exercise: Define a Basic Render Method (10 minutes)
```
Comment (amaseda) [inline 11-multi-track/bb_views_part1/readme.md]: Useful to reference code from the earlier OOJS lessons?

File: 11-multi-track/bb_views_part1/readme.md [hunk 3]
```diff
@@ -111,14 +118,14 @@ Just like in vanilla JS, we usually want to render HTML and put it into the
...
 
 Use the `.html()` method to fill your view's `el` with some basic HTML. For now,
 just make an `<h2>` and set it's content to be the grumble's title.
 
 Remember that each view has a `model` property.
 
-Test your view by instantiating one, calling `.render()`, and looking at the el:
+Test your view by instantiating one, calling `.render()`, and looking at the `el`...
```
Comment (amaseda) [inline 11-multi-track/bb_views_part1/readme.md]: Have solution ready to go (possibly link to in lesson plan).

File: 11-multi-track/bb_views_part1/readme.md [hunk 4]
```diff
@@ -16,45 +16,52 @@
...
+Like models, Backbone views have many similarities to the views we built in our hand-rolled OOJS classes. In Backbone, views are the objects responsible for...
+
+* Rendering content data into HTML and appending it to the page.
+* Listening to their models (for changes, destroys, etc.) and responding accordingly.
+* Listening for events on the page (usually within their `el`) and responding accordingly.
+
+They are sometimes called **View Controllers**, since they react to events as well as render data.
 
 Our views will always extend `Backbone.View` to ensure we get the behavior we
 want.
```
Comment (amaseda) [inline 11-multi-track/bb_views_part1/readme.md]: First take a look at the global `App` object in `app.js`.

File: 11-multi-track/bb_views_part1/readme.md [hunk 5]
```diff
@@ -16,45 +16,52 @@
...
+## Getting Started
 
-In Backbone, views are the objects responsible for:
+Today we will continue working on the Grumblr application we started building yesterday.  
 
-- Rendering content data into HTML, and appending it to the page
-- Listening to their models (for changes, destroys, etc), and responding accordingly
-- Listening for events on the page (usually within their `el`), and responding accordingly
+* [Starter Code](https://github.com/ga-dc/grumblr_backbone/tree/views_part_1_starter). Located on the `views_part_1_starter` branch.
+* [Solution Code](https://github.com/ga-dc/grumblr_backbone/tree/views_part_1_solution). Located on the `views_part_1_solution` branch.
```
Comment (amaseda) [inline 11-multi-track/bb_views_part1/readme.md]: Starter code `index.html` references comment files that do not exist. Either create or delete references.

File: 11-multi-track/bb_views_part1/readme.md [hunk 6]
```diff
@@ -172,14 +179,14 @@ And the in our index.html:
 
 ### Exercise: Define a Handlebars Template (10 minutes)
 
-1. Define a handlebars template for our grumble. (See the sample HTML below)
+1. Define a Handlebars template for our Grumble. See the sample HTML below.
```
Comment (amaseda) [inline 11-multi-track/bb_views_part1/readme.md]: Indicate that this needs to be placed as a separate script in `index.html` with `id=grumbleTemplate`.

File: 11-multi-track/bb_views_part1/readme.md [hunk 7]
```diff
@@ -172,14 +179,14 @@ And the in our index.html:
 
 ### Exercise: Define a Handlebars Template (10 minutes)
 
-1. Define a handlebars template for our grumble. (See the sample HTML below)
+1. Define a Handlebars template for our Grumble. See the sample HTML below.  
 2. On initialize, set a `this.template` property, which should point to a
-**compiled handlebars template**
+**compiled handlebars template**.
```
Comment (amaseda) [inline 11-multi-track/bb_views_part1/readme.md]: Provide syntax for `Handlebars.compile`.

File: 11-multi-track/bb_views_part1/readme.md [hunk 8]
```diff
@@ -172,14 +179,14 @@ And the in our index.html:
...
-1. Define a handlebars template for our grumble. (See the sample HTML below)
+1. Define a Handlebars template for our Grumble. See the sample HTML below.  
 2. On initialize, set a `this.template` property, which should point to a
-**compiled handlebars template**
+**compiled handlebars template**.  
 3. Update the `render` function to use the template. When using the template,
-we need to pass in just our model attributes, not the whole model, do some
-research to find the best way!
+we need to pass in just our model attributes - not the whole model. Do some
+research to find the best way!
```
Comment (amaseda) [inline 11-multi-track/bb_views_part1/readme.md]: So we need to pass `this.template` to `.html`. But what needs to be passed into `this.template`? (The model, but how do we write that out?)

File: 11-multi-track/bb_views_part1/readme.md [hunk 9]
```diff
@@ -172,14 +179,14 @@ And the in our index.html:
...
-**compiled handlebars template**
+**compiled handlebars template**.  
 3. Update the `render` function to use the template. When using the template,
-we need to pass in just our model attributes, not the whole model, do some
-research to find the best way!
+we need to pass in just our model attributes - not the whole model. Do some
+research to find the best way!  
 4. For good measure, let's call the `render()` method at the end of initialize,
-so we don't have to explicitly call it when we create new views.
+so we don't have to explicitly call it when we create new views.
```
Comment (amaseda) [inline 11-multi-track/bb_views_part1/readme.md]: Also need to add `.append` to `render()`

File: 11-multi-track/bb_views_part1/readme.md [hunk 10]
```diff
@@ -202,39 +209,37 @@ Sample HTML:
...
+Spend two minutes looking at the catalog of events in the above documentation.
 Which events do you think we might use most often, and why?
 
 ### Exercise: Re-render on Change (5 minutes)
 
 A very common pattern is to have a view listen to it's model for change events,
-and re-render in response. Take a moment to set this up. Hint: you'll only need
-to add one line of code to make this work.
+and re-render in response. Take a moment to set this up.  
+> Hint: You'll only need to add one line of code to make this work.
```
Comment (amaseda) [inline 11-multi-track/bb_views_part1/readme.md]: `this.listenTo(this.model, "change", this.render);`

File: 11-multi-track/bb_views_part1/readme.md [hunk 11]
```diff
@@ -247,10 +252,11 @@ var my_grumble = new App.Models.Grumble({
 var grumbleView = new App.Views.Grumble({model: my_grumble});
 $('#grumbles').append(grumbleView.$el)
 
-my_grumble.set('title', "Cats are the Best!"); // watch the page update automatically!
+my_grumble.set('title', "Cats are the Best!");
+// After running the above line, the page should update automatically!
 ```
 
-## Aside: Fetching, Rendering, and Auto-Appending (5 minutes)
+## Aside: Fetching, Rendering and Auto-Appending (5 minutes)
```
Comment (amaseda) [inline 11-multi-track/bb_views_part1/readme.md]: Now we can remove `.append` from `render()`

File: 11-multi-track/bb_views_part1/readme.md [hunk 12]
```diff
@@ -274,48 +280,49 @@ We'll see later that this sort of thing will be handled by a collection view.
...
-should:
+should...
 
-0. Prevent the default action of submitting the form (IMPORTANT!)
-1. Replace the `el` with a form, using a new template you'll have to define.
+1. Prevent the default action of submitting the form. **Very important!**    
+2. Replace the `el` with a form, using a new template you'll have to define.  
 
-The template should produce HTML that looks like this for the updated `el`:
+The template should produce HTML that looks like this for the updated `el`...
```
Comment (amaseda) [inline 11-multi-track/bb_views_part1/readme.md]: Get rid of `` from form.

File: 11-multi-track/bb_views_part1/readme.md [hunk 13]
```diff
@@ -337,21 +344,20 @@ When the user clicks the `update` button, call the `updateGrumble` method.
...
 
-0. Prevent the default action of submitting the form.
-1. Gather data from the form.
-2. Use the `save` method on the associated grumble (and the data from the form)
+1. Prevent the default action of submitting the form.
+2. Gather data from the form.
+3. Use the `save` method on the associated grumble (and the data from the form)
 to update the grumble.
-3. Re-render the show view.
+4. Re-render the show view.
```
Comment (amaseda) [inline 11-multi-track/bb_views_part1/readme.md]:
```
updateGrumble: function(){
    event.preventDefault();
    var data = {
      title: this.$("[name='title']").val(),
      authorName: this.$("[name='authorName']").val(),
      content: this.$("[name='content']").val(),
      photoUrl: this.$("[name='photoUrl']").val()
    };
    this.model.save(data);
  }
```

Why does this automatically hide the form?

File: 11-multi-track/bb_views_part1/readme.md [hunk 14]
```diff
@@ -337,21 +344,20 @@ When the user clicks the `update` button, call the `updateGrumble` method.
...
 
 ### Exercise: Cancel Button (5 minutes)
 
 When the user clicks the `cancel` button, go back to the show view.
```
Comment (amaseda) [inline 11-multi-track/bb_views_part1/readme.md]: `'click .cancel': 'render'`

File: 11-multi-track/bb_views_part1/readme.md [hunk 15]
```diff
@@ -16,45 +16,53 @@
 * [Backbone Tutorials](https://backbonetutorials.com)
 * [Backbone.js for absolute beginners](http://adrianmejia.com/blog/2012/09/11/backbone-dot-js-for-absolute-beginners-getting-started/)
 
-## Intro - What are Backbone Views? (10 minutes)
+## Getting Started
 
-In Backbone, views are the objects responsible for:
+Today we will continue working on the Grumblr application we started building yesterday.
```
Comment (amaseda) [inline 11-multi-track/bb_views_part1/readme.md]: Add bit about `http-server`

File: 11-multi-track/bb_views_part1/readme.md [hunk 16]
```diff
@@ -111,14 +136,15 @@ Just like in vanilla JS, we usually want to render HTML and put it into the
...
+### Exercise: Define a Basic Render Method (10 minutes)
 
 Use the `.html()` method to fill your view's `el` with some basic HTML. For now,
 just make an `<h2>` and set it's content to be the grumble's title.
+> Not sure how to begin? Think about how we defined `render` in our [earlier OOJS classes](https://github.com/ga-dc/curriculum/tree/master/08-single-page-web-apps/oojs1-read#views-artist-2090). Your Backbone render method in this exercise, however, will be much shorter.
 
 Remember that each view has a `model` property.
 
-Test your view by instantiating one, calling `.render()`, and looking at the el:
+Test your view by instantiating one, calling `.render()`, and looking at the `el`...
```
Comment (andrewsunglaekim) [inline 11-multi-track/bb_views_part1/readme.md]: might be good to have a defined render function here, or in some cliffnotes at the end.

**Pull request merged**